### PR TITLE
Minor bug fixes

### PR DESCRIPTION
**Pull request by BaBrixx**

### CODE REVIEW

**Review by atjn — APPROVED** [review]
I would like to thank you personally

File: src/main/java/dk/aau/p4/abaaja/Lib/Visitors/SymbolTableVisitor.java [hunk 1]
```diff
@@ -189,6 +189,11 @@ public void visit(IfStateNode node) {
                 );
             }
         }
+
+        // Type check the individual blocks
+        for (BlockNode block : node.get_blockChildrenNode()) {
+            visit(block);
+        }
```
Comment (Jesper276) [inline src/main/java/dk/aau/p4/abaaja/Lib/Visitors/SymbolTableVisitor.java:196]: stop reading here

**Review by NybyDK — APPROVED** [review]
HUGE! Thanks for your contribution!

**Review by Jesper276 — APPROVED** [review]
LGTM but did not read it all

**Review by TSKsmiley — APPROVED** [review]
omge

**Pull request merged**